solutions/cfgym-102694C.mdx

```diff
@@ -0,0 +1,245 @@
+---
+id: cfgym-102694C
+source: CF
+title: Sloth Naptime
+author: Sofia Yang
+---
+
+[Announcement With Official Editorial](https://codeforces.com/blog/entry/81527)
+
+**Time Complexity:** $\mathcal{O}(N \log N)$
+
+The sloth will always try to move upwards from the start node $\texttt{st}$ to
```

Comment:
Maybe explain the problem a little bit before jumping into the sol? IDK if that's needed or not though